packages/core/tsconfig.all.json

```diff
@@ -0,0 +1,5 @@
+{
+  "extends": "../../tsconfig.base.json",
+  "files": [],
+  "references": [{ "path": "./tsconfig.json" }, { "path": "./tsconfig.esm.json" }]
```

Comment:
Should we rename the `tsconfig.json` to `tsconfig.cjs.json`?

Reply:
we have to keep a `tsconfig.json` at root level I think, for tooling to find? Can add a `cjs` tho maybe that derives from `base`

Reply:
Actually I think current situation is ok. Nothing wrong with default being `cjs`, adding extra file does not really change anything